app/presenters/menu/default_menu.rb

```diff
@@ -186,16 +186,21 @@ def block_storage_menu_section
                                            'cloud_volume_type',
                                            {:feature => 'cloud_volume_type_show_list'},
                                            '/cloud_volume_type/show_list'),
-                            Menu::Item.new('storage_resource',
-                                           N_('Storage Resources'),
-                                           'storage_resource',
-                                           {:feature => 'storage_resource_show_list'},
-                                           '/storage_resource/show_list'),
+                            Menu::Item.new('host_initiator',
+                                           N_('Host Initiator'),
```

Comment:
this should be plural to be consistent with other menu items.

Reply:
done